gnat2goto/driver/goto_utils.ads

```diff
@@ -18,6 +18,15 @@ package GOTO_Utils is
    --  TODO change this to Irep
    function Internal_Source_Location return Source_Ptr;
 
+   function C_Float_T return Irep;
+   function C_Double_T return Irep;
+   function C_Int_T return Irep;
+   function C_Long_T return Irep;
```

Comment:
What are these for?

Reply:
Shortcuts, so that we don't create a new irep every time we refer to the integer type. Also it helps implementing the `Maybe_Double_Type_Width` function.

Reply:
Fair.